testing/features/step_definitions/navigation_steps.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+Given("the navigation is on the page") do
+  @component = Navigation::Default.new.tap(&:load)
+end
+
+Given("the navigation is on the page with header links") do 
+  @component = Navigation::WithHeader.new.tap(&:load)
+end
+
+Given("the navigation does not fit on the screen") do 
+  page.driver.browser.manage.window.resize_to(320, 568)
+end
+
+Given("the dropdown menu is already open") do
+  page.driver.browser.manage.window.resize_to(320, 568)
+  click_button("More")
+end
+  
+Then("the dropdown toggle is not present") do 
+  expect(@component).to have_no_button("More")
+end
+
+Then("the dropdown toggle is present") do 
+  expect(@component).to have_button("More")
+end
+
+Then("the dropdown menu is hidden") do
+  expect(@component).to have_no_link("More from us")
+end
+
+Then("the dropdown menu is present") do 
+  expect(@component).to have_link("More from us")
+end
+
+Then("the Close button is present") do
+  expect(@component).to have_button("Close")
+end
+
+Then("the More button is present") do
+  expect(@component).to have_button("More")
+end
+
+Then("the header links should be included in the dropdown menu") do
+  expect(@component).to have_button("Sign out")
+end
+
+When("I click the More button") do 
+  click_button("More")
+end
+
+When("I click the Close button") do
+  click_button("Close")
+end
+
+When("I tab onto the More button") do
+  5.times do
```

Comment:
feels a bit brittle

Reply:
We have a tab helper. It's in the header tests (Which caters to safari which is a bit odd).

It might not work directly out of the box. But if you can try "hook" into that. Maybe by providing an additional keyword arg or something - I'll let you run wild.

Reply:
Will potentially look at refactoring as it's tightly coupled to testing the skip links currently.